TPP.Core/Commands/Definitions/BadgeCommands.cs

```diff
@@ -253,18 +267,51 @@ public async Task<CommandResult> Pokedex(CommandContext context)
+                };
+            }
+            else if (_pokedexModeRegions.ContainsKey(mode.ToLower()))
+            {
+                Generation modeGeneration = _pokedexModeRegions[mode.ToLower()];
+
+                // Getting the total number of distinct Pokemon from a single generation could also be a function in class PokedexData,
+                //   by using "GenMaxIds", but I'm not sure how to get the total number of Fakemons
+                int totalRegionCount = _knownSpecies.Count(pokemon => pokemon.GetGeneration() == modeGeneration);
+                ImmutableSortedDictionary<PkmnSpecies, int> ownedPokemons = (await _badgeRepo.CountByUserPerSpecies(user.Id));
+                int userOwnedRegionCount = ownedPokemons.Count(ownedPokemon =>
+                    ownedPokemon.Key.GetGeneration() == modeGeneration);
+                return new CommandResult
+                {
+                    Response = isSelf
+                        ? $"You have collected {userOwnedRegionCount}/{totalRegionCount} "
+                          + $"distinct {System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(mode)} badge(s)"
+                        : $"{user.Name} has collected {userOwnedRegionCount}/{totalRegionCount} "
+                          + $"distinct {System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(mode)} badge(s)"
+                };
+            }
             else if (mode.Equals(PokedexModeModes))
             {
                 return new CommandResult
                 {
                     Response = $"Supported modes are '{PokedexModeDupes}': Show duplicate badges, '{PokedexModeMissing}': Show missing badges, "
                         + $"'{PokedexModeComplementFrom}' Compares missing badges from User A with owned badges from User B, "
-                        + $"'{PokedexModeComplementFromDupes}' Compares missing badges from User A with owned duplicate badges from User B"
+                        + $"'{PokedexModeComplementFromDupes}' Compares missing badges from User A with owned duplicate badges from User B, "
+                        + $"'{_pokedexModeRegions.ElementAt(0).Key}','{_pokedexModeRegions.ElementAt(1).Key}',... Shows how many badges you or the specified user owns from the specified region only"
                 };
             }
             return new CommandResult
             {
-                Response = $"Unsupported mode '{mode}'. Current modes supported: {PokedexModeModes}, {PokedexModeDupes}, {PokedexModeMissing}, {PokedexModeComplementFromDupes}, {PokedexModeComplementFrom}"
+                Response = $"Unsupported mode '{mode}'. Current modes supported: {PokedexModeModes}, {PokedexModeDupes}, {PokedexModeMissing}, {PokedexModeComplementFromDupes}, "
+                    + $"{PokedexModeComplementFrom}, {_pokedexModeRegions.ElementAt(0).Key}, {_pokedexModeRegions.ElementAt(1).Key},... "
```

Comment:
I believe using `ElementAt` with a non-sorted dictionary will cause this command to effectively print two random region names. The same goes for other occurances of `ElementAt`

Reply:
I guess the attempt to make this as generic as possible was too sophisticated anyway. The chances that a region gets renamed are near 0, so I might as well just add them as hardcoded examples here.